code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -143,7 +143,7 @@
 
 		finish_setup(new_character, i)
 		assigned += applicant
-		notify_ghosts("[new_character] has been picked for the ruleset [name]!", source = new_character, action = NOTIFY_ORBIT, header="Something Interesting!")
+		notify_ghosts("[ADMIN_LOOKUPFLW(new_character)] has been picked for the ruleset [name]!", source = new_character, action = NOTIFY_ORBIT, header="Something Interesting!")
```

Comment:
![image](https://user-images.githubusercontent.com/85910816/170331899-e759dc43-ca7b-4483-83e5-a642befa2c1a.png)

it does, since new_character is the mob 

and it's used fine in other places mob vars are used with it like below 

![image](https://user-images.githubusercontent.com/85910816/170333020-ff2a6e2f-d344-4e8b-b0f9-b8304974a697.png)

Reply:
> Are you sure this fixes that issue? `new_character` will still be "Space Dragon" or "Alien Larva", which will print plainly into the chat, and not print the ckey of the mob
> 
> changing it to `applicant` would get what you desire, I think

it's hard to test solo to be honest :))

Reply:
I've tried to run a local game many time and have only been able to via the tgstation.dmb file after the build file, which directly launches offline mode. So I can't actually test this myself without it being test merged or something.  Since you say this is best though I do know you're good at this and trust you. :)